spec/integration/hawkular_client_spec.rb

```diff
@@ -50,24 +60,28 @@ module Hawkular::Client::RSpec
       expect { @hawkular_client.tokens_lyst_feeds }.to raise_error(NoMethodError)
     end
 
-    context 'and URIs as input', vcr: { decode_compressed_response: true } do
+    context 'and URIs as input' do
       it 'Should work with URI' do
```

Comment:
My fault, I'll remove the commit that fixes the tests because I did another PR https://github.com/hawkular/hawkular-client-ruby/pull/208 for that,  put your comments on that PR please :), but without that fix, the hawkular client tests fails.